packages/block-editor/src/components/block-list/style.scss

```diff
@@ -424,6 +424,9 @@
 /**
  * Block Toolbar when contextual.
  */
+.block-editor-block-contextual-toolbar-wrapper {
+	padding-left: $grid-unit-60; // Provide space for the mover control on full-wide items.
```

Comment:
A better variable name here is `$block-toolbar-height` as that's the width the icon buttons are assigned. It's not the only place where this is used directly in G2 but I believe we should change all of these sizes (that related to the toolbar height) with the `$block-toolbar-height` variable instead. Because that way, if we change the toolbar height, it changes all the related sizes accordingly.